apps/web/.env.production.example

```
@@ -0,0 +1,8 @@
+NEXT_PUBLIC_SITE_URL=https://www.infamousfreight.com
+NEXT_PUBLIC_API_URL=https://api.infamousfreight.com/api
+NEXT_PUBLIC_APP_ENV=production
+NEXT_PUBLIC_APP_NAME=Infamous Freight
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_NAME key should go before the NEXT_PUBLIC_SITE_URL key

(UnorderedKey)